src/util/linkify.js

```diff
@@ -3,43 +3,50 @@ import linkifyString from 'linkifyjs/string'
 import cheerio from 'cheerio'
 import { isEmpty, toPairs } from 'lodash'
 
+export function hashtagHref (tagName, slug) {
+  return slug
+  ? `/c/${slug}/tag/${tagName}`
+  : `/tag/${tagName}`
+}
+
 // this handles old-style hashtags, which aren't wrapped in tags
 var linkifyjsOptions = {
   formatHref: function (value, type) {
     if (type === 'hashtag') {
-      return '/search?q=%23' + value.substring(1)
+      return '/tag/' + value.substring(1)
```

Comment:
you have to define the options inside `linkify` now instead of having them be constant.